dist/schema.json

```diff
@@ -82,7 +82,18 @@
             ]
           }
         ],
-        "parameters": [],
+        "parameters": [
+          {
+            "name": "Authorization",
```

Comment:
Exclude the `Authorization` header from the header parsing. It's already covered by the oauth parsing.